### ISSUE TITLE
Unhandled Promise Rejection

### ISSUE THREAD

**Issue by mitschmidt** [kucherenko/jscpd#185]
**Describe the bug**
When attempting to test the clone detection in my project, I ran `jscpd`, looked at the report and created a duplicate of one of my typescript files (obviously this is code cloning). However, when running `jscpd` afterwards I am running into this error without a report:

```
(node:911) UnhandledPromiseRejectionWarning: #<Object>
(node:911) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:911) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**To Reproduce**
Duplicated a `*.ts` file in my repo, which should yield a new  / altered clone report in jscpd.

My `.jscpd.json` looks like this:

```{
  "reporters": [
    "console",
    "xml",
    "html"
  ],
  "blame": true,
  "threshold": 2.27,
  "format": [
    "javascript",
    "typescript",
    "markup",
    "css",
    "scss"
  ],
  "ignore": [
    "**/node_modules/**",
    "**/reports/**",
    "**/dist/**",
    "**/shared/build/**",
    "**/docs/**",
    "**/client/app/shared/models/**",
    "**/server/api/routes/**",
    "**/server/database/models/**",
    "**/server/email/templates/**",
    "**/dbcli/src/databases/remote/seeders/**",
    "**/*.spec.ts"
  ],
  "output": "./reports/clone-detection/test-results/"
}
```

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Mac OS X
 - OS Version Mac OS
 - NodeJS Version 10.14

**Comment by kucherenko** [comment]
thank you for the report, will analyze and fix it

**Comment by kucherenko** [comment]
do you have any stake trace attached to the error?

**Comment by mitschmidt** [comment]
Unfortunately not - I re-executed using the `verbose`reporter but the same error output has been generated. Do you have any advise on how to generate a more informative error output?

**Comment by kucherenko** [comment]
Looks like it was a bug with blamer, fixed and published new version of jscpd - 1.2.1

**Comment by mitschmidt** [comment]
Wow, that was quick fix! Thank you and keep up the good work 👍
